examples/c/wind_turbine_nmpc.c

```diff
@@ -686,7 +785,6 @@ int main()
 		pcond_solver_opts->pcond_opts->N2 = 10;
 	}
 
-	// update after user-defined opts
```

Comment:
why removed?

Reply:
replaced this with another comment a bit below
``	/************************************************
	* 	precomputation (after all options are set)
	************************************************/ ``